backend/lambda_functions/Returns_Google_drive_file_contents/Returns_Google_drive_file_contents_lambda.py

```diff
@@ -0,0 +1,66 @@
+import json
+import boto3
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+from googleapiclient.http import MediaIoBaseDownload
+import io
+
+def get_secret():
+    secret_name = "google_drive_service_account"  # Replace with the name of your secret
+    region_name = "us-east-1"   # Replace with your AWS region
+
+    # Create a Secrets Manager client
+    session = boto3.session.Session()
```

Comment:
if you are not passing any arguments to the Session method, you might as well just use the `boto3.client` method to create the session.